sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -89,7 +89,9 @@ int pollSocketUntilEventOrTimeout(
 // Windows needs this after every write to socket or peformance would be reduced to 1/4 for
 // uploading operation.
 // https://github.com/Azure/azure-sdk-for-cpp/issues/644
-void WinSocketSetBuffSize(curl_socket_t socket, std::function<void(std::string const& message)> logger)
+void WinSocketSetBuffSize(
```

Comment:
The changes to this file are cosmetic **only** correct?

Reply:
Yes, auto-formatting